stable/kibana/values.yaml

```diff
@@ -155,6 +155,25 @@ dashboardImport:
   dashboards: {}
     # k8s: https://raw.githubusercontent.com/monotek/kibana-dashboards/master/k8s-fluentd-elasticsearch.json
 
+objectsImport:
+  enabled: false
+  timeout: 60
+  xpackauth:
+    enabled: false
+    username: myuser
```

Comment:
Let's comment theses in the values.yaml since they're only utilized if `enabled: true`